### PR TITLE
Fix/ola

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.html [hunk 1]
```diff
@@ -216,13 +213,9 @@
                 </thead>
                 <tbody>
                     <tr class="oui-datagrid__row"
-                        data-ng-repeat="interface in $ctrl.fakeInterfaces track by interface.name">
-                        <td class="oui-datagrid__cell">
-                            <span data-ng-if="interface.isPublic()" data-translate="dedicated_server_interfaces_type_public_label"></span>
-                            <span data-ng-if="interface.isPrivate()" data-translate="dedicated_server_interfaces_type_private_label"></span>
-                        </td>
+                        data-ng-repeat="mac in $ctrl.networkInterfaces track by mac">
```
Comment (FredericEspiau) [inline client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.html]:
```suggestion
                        data-ng-repeat="mac in ::$ctrl.networkInterfaces track by mac">
```

File: client/app/dedicated/server/bandwidth-legacy/order-private/dedicated-server-bandwidth-vrack-order.controller.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 class ServerOrderLegacyBandwidthVrackCtrl {
   /* @ngInject */
-  constructor($scope, $stateParams, $translate, User, BandwidthVrackOrderService) {
+  constructor($scope, $state, $stateParams, $translate, Alerter, User, BandwidthVrackOrderService) {
```
Comment (JeremyDec) [inline client/app/dedicated/server/bandwidth-legacy/order-private/dedicated-server-bandwidth-vrack-order.controller.js]:
```suggestion
  constructor($scope, $state, $stateParams, $translate, Alerter, BandwidthVrackOrderService, User) {
```

File: client/app/dedicated/server/bandwidth-legacy/order-public/dedicated-server-bandwidth-order.controller.js [hunk 3]
```diff
@@ -148,10 +148,8 @@ angular.module('App').controller('ServerOrderLegacyBandwidthCtrl', ($scope, $sta
 
         bcUrl = durations.url;
       },
-      (data) => {
-        $scope.resetAction();
-        _.set(data, 'data.type', 'ERROR');
-        $scope.setMessage($translate.instant('server_order_bandwidth_error'), data.data);
+      () => {
```
Comment (JeremyDec) [inline client/app/dedicated/server/bandwidth-legacy/order-public/dedicated-server-bandwidth-order.controller.js]: catch ?

File: client/app/dedicated/server/bandwidth/dashboard/dashboard.component.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import controller from './dashboard.controller';
+import template from './dashboard.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
```
Comment (JeremyDec) [inline client/app/dedicated/server/bandwidth/dashboard/dashboard.component.js]: Alphabetical order ?

File: client/app/dedicated/server/bandwidth/dashboard/dashboard.html [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        <div class="row">
+            <div class="col-md-6">
+                <!-- Outgoing Bandwidth -->
+                <span class="oui-icon oui-icon-arrow-up font-inherit"></span>
+                <span data-ng-bind="$ctrl.specifications.bandwidth.OvhToInternet | ducBandwidth"></span>
+                <span data-translate="dedicated_server_bandwidth_outgoing"></span>
+                <div class="text-danger" data-ng-if="$ctrl.specifications.traffic.usage.isThrottled">
+                    -
+                    <span data-translate="server_consumption_throttled"></span>
+                    <button type="button" class="oui-popover-button" oui-popover="{{:: 'server_consumption_explanation' | translate }}"></button>
```
Comment (JeremyDec) [inline client/app/dedicated/server/bandwidth/dashboard/dashboard.html]:
```suggestion
                    <button type="button" class="oui-popover-button" data-oui-popover="{{:: 'server_consumption_explanation' | translate }}"></button>
```

File: client/app/dedicated/server/bandwidth/dashboard/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  "dedicated_server_bandwidth_type_platinum": "Platinium",
+  "dedicated_server_bandwidth_type_premium": "Premium",
+  "dedicated_server_bandwidth_type_standard": "Standard",
+  "dedicated_server_bandwidth_type_ultimate": "Ultimate",
+  "dedicated_server_bandwidth_ovh_to_internet": "BP sortante",
+  "dedicated_server_bandwidth_internet_to_ovh": "BP entrante",
+  "dedicated_server_bandwidth_order_button": "Commander de la bande passante publique",
+  "dedicated_server_bandwidth_cancel_button": "Resilier",
+  "dedicated_server_bandwidth_option_released": "(non renouvelé)",
+  "dedicated_server_bandwidth_loading_error": "Une erreur est survenue lors des chargements des informations.",
```
Comment (JeremyDec) [inline client/app/dedicated/server/bandwidth/dashboard/translations/Messages_fr_FR.json]: ?
```suggestion
  "dedicated_server_bandwidth_loading_error": "Une erreur est survenue lors du chargement des informations.",
```

File: client/app/dedicated/server/dedicated-server.routes.js [hunk 7]
```diff
@@ -25,6 +39,39 @@ angular.module('App').config(($stateProvider) => {
...
+      resiliatePrivateBandwidthLink: /* @ngInject */
+        (
+          $state,
+          serverName,
+        ) => $state.href('app.dedicated.server.dashboard.bandwidth-private-cancel', { productId: serverName }),
+      resiliatePublicBandwidthLink: /* @ngInject */
+      (
+        $state,
+        serverName,
+      ) => $state.href('app.dedicated.server.dashboard.bandwidth-public-cancel', { productId: serverName }),
```
Comment (JeremyDec) [inline client/app/dedicated/server/dedicated-server.routes.js]: Uniformity of parethesis ?

File: client/app/dedicated/server/interfaces/interfaces.component.js [hunk 8]
```diff
@@ -5,11 +5,20 @@ export default {
...
     interfaces: '<',
     ola: '<',
-    urls: '<',
+    orderPrivateBandwidthLink: '<',
+    orderPublicBandwidthLink: '<',
+    server: '<',
+    serverName: '<',
+    specifications: '<',
+    resiliatePublicBandwidthLink: '<',
+    resiliatePrivateBandwidthLink: '<',
```
Comment (JeremyDec) [inline client/app/dedicated/server/interfaces/interfaces.component.js]: Alphabetical order

File: client/app/dedicated/server/interfaces/interfaces.routing.js [hunk 9]
```diff
@@ -11,21 +11,54 @@ export default /* @ngInject */ ($stateProvider) => {
...
+      resiliatePrivateBandwidthLink: /* @ngInject */
+        (
+          $state,
+          serverName,
+        ) => $state.href('app.dedicated.server.interfaces.bandwidth-private-cancel', { productId: serverName }),
+      resiliatePublicBandwidthLink: /* @ngInject */
+      (
+        $state,
+        serverName,
+      ) => $state.href('app.dedicated.server.interfaces.bandwidth-public-cancel', { productId: serverName }),
```
Comment (JeremyDec) [inline client/app/dedicated/server/interfaces/interfaces.routing.js]: Same for ()

File: client/app/dedicated/server/bandwidth-legacy/order-public/dedicated-server-bandwidth-order.controller.js (hunk 3 above)
Comment (marie-j) [reply client/app/dedicated/server/bandwidth-legacy/order-public/dedicated-server-bandwidth-order.controller.js]: I tried to limit the changes to this legacy component to as few as possible, but I keep the refactoring in mind

**Pull request merged**